README.md

```diff
@@ -1,2 +1,58 @@
 # litter-app
 Facilitating the planning and organisation of fly tip cleanups
+
+[Link to prototype](https://github.com/JWLD/litter-locator)
+
+#### User Journeys:
+
+- People can document litter sites (ie fly tipping, build-ups of illegal litter, or undesirable waste located in public areas), and report details of its size, location, and type.
+
+- Volunteers and concerned citizens can then browse documented litter sites by location using map UIs, and view the litter sites' details in order to plan cleanups to remove the waste.
+
+#### User Stories
```

Comment:
The formatting is weird with these user stories - got a bullet point halfway down each one